### PR TITLE
#424 : Add licensed images label to licensed images thumbnail in the grid 

### PR DESCRIPTION
**Pull request by konarshankar07**
### Description (*)

This PR will introduce the label for the licensed images in the grid
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#424: Add licensed images label to licensed images thumbnail in the grid 

### Preconditions
Delete data with namespace `adobe_stock_images_listing` from ` ui_bookmarks` table

1. Login to admin panel
2. Open Magento Media Gallery (i.e. go to Cms -> Pages, edit the page and insert image)
3. Click "Search Adobe Stock" button to open images grid
4. Sign in to Adobe Stock (via Sign-in link or by clicking License image button from image preview)
5. Search for an image that is licensed for the account used to sign-in

### CODE REVIEW

**Comment by konarshankar07** [comment]
Hello @sivaschenko ,
Can you please check the logic for getting `is_licensed`? Am I on right track?
Thanks

**Comment by sivaschenko** [comment]
Blocked by SDK issue: https://github.com/adobe/stock-api-libphp/pull/10

File: AdobeUi/view/adminhtml/web/js/components/grid/column/overlay.js [hunk 1]
```diff
@@ -25,6 +25,15 @@ define([
             }
 
             return {};
+        },
+
+        isLicensed: function (record) {
```
Comment (sivaschenko) [inline AdobeUi/view/adminhtml/web/js/components/grid/column/overlay.js]: Please extend overlay from AdobeUi module in AdobeStockImageAdminUi module and introduce logic related to Adobe Stock there (same for js file and template).

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @konarshankar07 thanks for working on this task! 

The issue with `is_licensed` value was fixed in https://github.com/adobe/stock-api-libphp/pull/10 - please update your sdk version running `composer require astock/stock-api-libphp:1.1.2`

Please do not add Adobe Stock logic to AdobeUi module (this module contains generic UI components that will be delivered to magento2 repo), see my review comments.

**Review by sivaschenko — APPROVED** [review]

**Comment by konarshankar07** [comment]
Hello @sivaschenko ,
I think you forgot to check that `is_licensed` is need to be in result_column if there is an access token otherwise it will throw the exception. Correct me if I'm wrong
Thanks

**Comment by konarshankar07** [comment]
I'm very close to finishing this issue and it might get finished by today. Do I need to create another PR?

**Comment by sivaschenko** [comment]
@konarshankar07 yes, please create another PR.

**Pull request merged**